index.js

```diff
@@ -1788,3 +1783,17 @@ function numberIsNaN (obj) {
   // For IE11 support
   return obj !== obj // eslint-disable-line no-self-compare
 }
+
+// Create lookup table for `toString('hex')`
+// See: https://github.com/feross/buffer/issues/219
+var hexSliceNewLookupTable = (function () {
```

Comment:
Why this variable name? Would something simpler like `hexLookupTable` work?

Reply:
Wanted variable which will be associated with function where used, but it's not important for me. If `hexLookupTable` better fit for you I'll send fix.

Reply:
Since the function is called `hexSlice` I think `hexSliceLookupTable` is a good name.

Reply:
Oh, just realized that `New` is extra here. Copy-pasta from benchmarks =\
Sorry!